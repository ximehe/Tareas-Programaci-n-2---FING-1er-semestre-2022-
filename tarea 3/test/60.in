# implementar buscarFinPrefijo
insertarPalabra al
insertarPalabra cal
insertarPalabra can
insertarPalabra la
insertarPalabra los
insertarPalabra losas
insertarPalabra mensaje
buscarFinPrefijo c
buscarFinPrefijo m
buscarFinPrefijo lo
buscarFinPrefijo losas
buscarFinPrefijo q
buscarFinPrefijo mensaj
reiniciarEstructuras
insertarPalabra a
insertarPalabra mensaje
insertarPalabra mes
insertarPalabra mesa
insertarPalabra mesada
buscarFinPrefijo a
buscarFinPrefijo men
buscarFinPrefijo mena
buscarFinPrefijo mes
Fin