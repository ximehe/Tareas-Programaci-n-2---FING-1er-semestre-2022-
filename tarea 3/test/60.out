1>#  implementar buscarFinPrefijo.
2>Insertando al.
3>Insertando cal.
4>Insertando can.
5>Insertando la.
6>Insertando los.
7>Insertando losas.
8>Insertando mensaje.
9>
cal
can
10>
mensaje
11>
los
losas
12>
losas
13>
palabras no contiene q.
14>
mensaje
15>Estructuras reiniciadas.
16>Insertando a.
17>Insertando mensaje.
18>Insertando mes.
19>Insertando mesa.
20>Insertando mesada.
21>
a
22>
mensaje
23>
palabras no contiene mena.
24>
mes
mesa
mesada
25>Fin.
